packages/editor/src/pages/Editor/store/samples/selectors.ts

```diff
@@ -1,7 +1,7 @@
 import { IState } from '../reducer';
 
 export const getMetadataByGroup = (state: IState): ISampleMetadataByGroup =>
-  Object.values(state.samples)
+  Object.values(state.samples || [])
```

Comment:
shouldn't this be `|| {}`?

Reply:
Doesn't matter, they produce the same thing.  But yeah, I can change to `{}`.  When I first read it I misread it and thought that Samples was an array rather than a dictionary...

Reply:
but by that logic, so does: Object.values(() => {})... still clearer to write {} I think though